package-lock.json

```diff
@@ -36,7 +36,7 @@
         "tape-catch": "^1.0.6"
       },
       "engines": {
-        "node": ">=4.6.2"
+        "node": "^16 || ^14 || ^12"
```

Comment:
Can you also update the lockfile? An `npm install` should do it for you.

Reply:
sure thing, sorry I forgot:+1:  PR updated